.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+// {
+// 	"go.toolsManagement.checkForUpdates": "local",
```

Comment:
this is looking all commented/wrong? maybe an oversight?

Reply:
Hi, @mudler . It is hard to have a dynamic configuration for all the environments. This configuration only working for all the Codespace environment, but it will not support the local environment. So, I need to commend all, make sure the contributor without getting any issue when they are using VSCode in their local environment.

Reply:
I think it's confusing then to have a commented json file in the code. I'd avoid the confusion then and do not commit it at all, maybe we can document it separately tho

Reply:
Will do.